RecoPixelVertexing/PixelTriplets/python/pixelTripletHLTEDProducer_cff.py

```diff
@@ -0,0 +1,7 @@
+from RecoPixelVertexing.PixelTriplets.pixelTripletHLTEDProducer_cfi import *
+from Configuration.Eras.Modifier_trackingLowPU_cff import trackingLowPU
+from Configuration.Eras.Modifier_trackingPhase1PU70_cff import trackingPhase1PU70
+from Configuration.Eras.Modifier_trackingPhase2PU140_cff import trackingPhase2PU140
+trackingLowPU.toModify(pixelTripletHLTEDProducer, maxElement=100000)
+trackingPhase1PU70.toModify(pixelTripletHLTEDProducer, maxElement=0)
+trackingPhase2PU140.toModify(pixelTripletHLTEDProducer, maxElement=0)
```

Comment:
What I can do is to change `fillDescriptions` to generate `pixelTripletHLTEDProducerDefault_cfi`, rename this file to `_cfi`, and do
```
pixelTripletHLTProducer = pixelTripletHLTEDProducerDefault.clone()
```
before the era modifications. Would that be sufficient?